vita3k/gui/src/live_area.cpp

```diff
@@ -1013,7 +1014,7 @@ void draw_live_area_screen(GuiState &gui, EmuEnvState &emuenv) {
         const auto update_pos = ImVec2((manual_exist ? 408.f : 463.f) * SCALE.x, 505.0f * SCALE.y);
         const auto pos_scal_update = ImVec2(display_size.x - update_pos.x, display_size.y - update_pos.y);
 
-        const auto UPDATE_STR = "Update";
+        const auto UPDATE_STR = gui.lang.app_context["update"].c_str();
```

Comment:
Revert all buttons, i keep it in english only for good reason 🙏 thx

Reply:
got it.